src/services/team-member-api.js

```diff
@@ -1,4 +1,8 @@
-const apiUrl = 'http://localhost:3000/team-members';
+const apiUrl = `${
```

Comment:
You're not restoring the server-side sorting?

Reply:
This happens specifically under `fetchTeamMembers`